webapp/packages/core-settings/src/createDeprecatedResolver.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { ISyncExecutor, SyncExecutor } from '@cloudbeaver/core-executor';
+import type { schema } from '@cloudbeaver/core-utils';
+
+import type { ISettingChangeData, ISettingsSource } from './ISettingsSource';
+import type { SettingsProvider } from './SettingsProvider';
+
+const DEPRECATED_SETTINGS = new Set();
+
+type SettingsMapping<TTarget> = Partial<{
+  [key in keyof TTarget]: string;
+}>;
+
+export function createSettingsAliasResolver<TTarget extends schema.SomeZodObject>(
```

Comment:
please mark this as deprecated

Reply:
this function is helper and it's not deprecated by itself (we don't want to remove it)

but we use it for supporting deprecated settings reading